dashboard/app/controllers/projects_controller.rb

```diff
@@ -15,4 +21,16 @@ def template
 
     render template: "projects/#{params[:template]}", layout: nil
   end
+
+  def new
```

Comment:
Can I pass the `@is_project` instance var to that action?  When we had a similar situation in `embed_level` (needed to set some extra instance vars) we opted for a separate action for clarity.